src/components/Collective.vue

```diff
@@ -219,6 +221,9 @@ export default {
 		async getAllPages(setLoading = true) {
 			await this.getPages(setLoading)
 				.catch(displayError('Could not fetch pages'))
+			await this.getTags()
+				.catch(displayError('Could not fetch tags'))
+			await this.getTags()
```

Comment:
Seems like a typo - I don't see why you'd want to get them twice.

Reply:
Yes, indeed. But already fixed in a subsequent commit :face_with_peeking_eye: